docs/README.md

```diff
@@ -607,6 +609,61 @@ The easiest way to get started with production-grade metrics collection is to us
 
 Congratulations! You should now be able to see your control plane and data plane metrics in the OpenShift Console. Just go to Observe -> Metrics and try the query `istio_requests_total`.
 
+### Configure tracing with OpenShift distributed tracing
+This section describes how to setup Istio with OpenShift Distributed Tracing to send distributed traces.
+
+*Prerequisites*
+* A Tempo stack is installed and configured
+* An instance of an OpenTelemetry collector is already configured in the istio-system namespace
+* An Istio instance is created with the OpenShift profile
+* An Istio CNI instance is created with the OpenShift profile
+
+*Steps*
+1. Configure Istio to enable tracing and include the OpenTelemetry settings:
+    ```yaml
+    meshConfig:
+      enableTracing: true
+      extensionProviders:
+      - name: otel-tracing
+        opentelemetry:
+          port: 4317
+          service: otel-collector.istio-system.svc.cluster.local 
+    ```
+The *service* field is the OpenTelemetry collector service in the `istio-system` namespace.
+
+2. Create an Istio telemetry resource to active the OpenTelemetry tracer
+    ```yaml
+    apiVersion: telemetry.istio.io/v1
+    kind: Telemetry
+    metadata:
+      name: otel-demo
+      namespace: istio-system
+    spec:
+      tracing:
+      - providers:
+          - name: otel-tracing
+            randomSamplingPercentage: 100
+    ```
+
+3. Validate the integration: Generate some traffic
+
+We can deploy bookinfo and generate some traffic:
+
+```sh
+kubectl create ns bookinfo
+kubectl label ns bookinfo istio-injection=enabled
```

Comment:
These steps don't actually deploy bookinfo? I think elsewhere we already describe how to deploy bookinfo. Maybe you can just link to that and assume here that you've already deployed it?

Reply:
Yes, right, it is linked now.